packages/server/modules/workspacesCore/domain/types.ts

```diff
@@ -1,5 +1,24 @@
+import { InviteResourceTarget } from '@/modules/serverinvites/domain/types'
```

Comment:
moved some constants & types to workspacesCore, since I needed to reference them outside of workspaces